### PR TITLE
Twitter - Tidying up Spice code (fixes #1931)

### LINKED ISSUE #1931: Twitter – Tidying up Spice code (template, trigger fails, latest tweet functionality)
The Twitter spice code looks rather messy.  Don't mind working on this, but thought I should consolidate some of the outstanding issues here for ease of reference.

- Switch to Icon Template Group – #1864
- Switch to latest Spice API – not in #1896 but should be referenced as well
- Results not showing latest tweet – Is it supposed to?  It only seems to be showing the account description and not the tweet.  If so, then description on IA page should be changed too.

Anything else to add?

Can someone also explain what `https://duckduckgo.com/tw.js` is?

***
IA Page: https://duck.co/ia/view/twitter

### PR DESCRIPTION
**Pull request by altern8tif**
- Switch to Icon Template Group – #1864
- Switch to latest Spice API – #1896
- Update IA metadata description
- Remove CSS and Handlebar files

### CODE REVIEW

**Comment by altern8tif** [comment]
screen shot 2015-06-18 at 11 27 48 pm

**Comment by altern8tif** [comment]
Addresses #1931

File: share/spice/twitter/twitter.js [hunk 1]
```diff
@@ -1,132 +1,59 @@
...
+        function getURL(item) {
+            var url = DDG.getProperty(item, 'entities.url.urls.0.url');
+            var text = DDG.getProperty(item, 'entities.url.urls.0.display_url');
+            if (url) {
+                return [{
+                   text: text,
+                   href: url
+                }];
+            } else {
+                return "No website provided";
```
Comment (moollaza) [inline share/spice/twitter/twitter.js]: Probably best to simply not show anything if a website isn't given

Comment (moollaza) [inline share/spice/twitter/twitter.js]: Meaning, don't say that there's nothing, the lack of it being there should signal that ;)

**Comment by moollaza** [comment]
We can fix the `no website` in another PR. Would like to get this merged :+1:

**Comment by altern8tif** [comment]
@moollaza sure thing, will do a new PR soon... Hadn't had the time to amend it. 

Thanks for the merge!

**Pull request merged**